packages/twenty-front/src/modules/object-record/record-index/export/hooks/useExportRecords.ts

```diff
@@ -107,12 +107,9 @@ const percentage = (part: number, whole: number): number => {
   return Math.round((part / whole) * 100);
 };
 
-export const displayedExportProgress = (
-  mode: 'all' | 'selection' = 'all',
-  progress?: ExportProgress,
-): string => {
+export const displayedExportProgress = (progress?: ExportProgress): string => {
   if (isUndefinedOrNull(progress?.exportedRecordCount)) {
-    return mode === 'all' ? 'Export View as CSV' : 'Export Selection as CSV';
+    return 'Export';
```

Comment:
why are we getting rid of this behavior?

Reply:
It's like that on Figma, so I've change it

<img width="283" alt="Screenshot 2024-11-19 at 8 57 58 AM" src="https://github.com/user-attachments/assets/0d94e615-d7b1-4de6-a6da-bb781c034001">